src/elexmodel/models/BaseElectionModel.py

```diff
@@ -264,3 +266,36 @@ def get_coefficients(self):
         These coefficients are for the point prediciton only, not for the lower or upper intervals models.
         """
         return self.features_to_coefficients
+
+    def compute_lambda(
+        self,
+        reporting_units,
+        possible_lambda_values=[],
+        estimand="",
+        K=3,
+    ):
+        if len(possible_lambda_values) == 0:
```

Comment:
if `possible_lambda_values` just has one element in it, then I don't think we would want to run `compute_lambda` either, right? Since we know what the optimal lambda would be from that set.